packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-out/main/aspnet/summaryofsteps.md

```diff
@@ -2,6 +2,10 @@ This use case signs a user out of the local app by revoking their session tokens
 
 <div class="three-quarter">
 
-![Basic sign-out sequence diagram](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-seq.png)
+![A flow diagram showing the interactions in a basic sign-out flow between user, client application, server-side SDK and Okta using the embedded ASP.NET SDK](/img/oie-embedded-sdk/oie-embedded-dotnet-sign-out-flow-diagram.png)
```

Comment:
```suggestion
![A flow diagram showing the interactions in a basic sign-out flow between user, client application, server-side SDK, and Okta using the embedded ASP.NET SDK](/img/oie-embedded-sdk/oie-embedded-dotnet-sign-out-flow-diagram.png)
```

Reply:
If accepted, match changes to the other files as well.

Reply:
Sure, I can add that comma. Will update the rest too.